pkg/logs/client/http/destination.go

```diff
@@ -218,11 +220,11 @@ func (d *Destination) sendAndRetry(payload *message.Payload, output chan *messag
 		if err != nil {
 			metrics.DestinationErrors.Add(1)
 			metrics.TlmDestinationErrors.Inc()
+			log.Warnf("Could not send payload: %v", err)
```

Comment:
was this log line moved up intentionally? (seems fine, just wondering)

Reply:
yeah, previously we were only logging if we got a context canceled error specifically; now we'll log any time we see an error.

Reply:
I wonder how often we even see errors that aren't timeouts 🤷 

hopefully never